src/lib/components/form.svelte

```diff
@@ -2,17 +2,19 @@
     import { FieldType } from '$lib/app'
     import type { Input as InputType, SubmitButton } from '$lib/app/types/form'
     import { Multiselect } from '$lib/components'
-    import { Button, FormGroup, Label, Spinner } from 'sveltestrap'
+    import { Button, FormGroup, Label, Spinner, Tooltip } from 'sveltestrap'
     import Input from 'sveltestrap/src/Input.svelte'
 
     export let enableValidation: boolean = false
     export let inputs: InputType[]
     export let onSubmitButton: SubmitButton
 
-    let inputFields = {}
     let unsubscribe
     let formValidated: boolean = false
     let formContainer: HTMLFormElement
+    let hidden: boolean = false
```

Comment:
I guess this won't work with multiple checkboxes in one form since they all would share the same state. Maybe something like a `defaultState` parameter in the DEFAULT_IDENTITIES_TEMPLATES object in `identity.ts` could help you.

Reply:
Also don't create variables specific to some form here. The `form.svelte` should be a general purpose component, reusable wherever a form is needed.